README.Mac.md

```diff
@@ -1,294 +1,285 @@
+[Sequel Pro](https://sequelpro.com/test-builds) (free)
 
-Ensure that the line is uncommented (remove the ```#``` at the beginning of the line)
+[TablePlus](https://tableplus.io) (free trial or pay for extra features)
 
-### 3.5.2 Adding new vhost
-Open the ```/etc/apache2/extra/httpd-vhosts.conf``` in the text editor and add the following:
+## 1.9 Install Composer
+Type into your terminal and hit enter:
+```
+brew install composer
 ```
-<VirtualHost *:80>
-	ServerAdmin webmaster@localhost
-	DocumentRoot "/Users/$username/Sites/loris/htdocs/"
-	ServerName localhost.loris
-	ErrorLog "/private/var/log/apache2/loris.com-error_log"
-	CustomLog "/private/var/log/apache2/loris.com-access_log" common
-	
-	<Directory "/Users/$username/Sites/loris/htdocs/">
-		Options Indexes FollowSymLinks
-		AllowOverride All
-		Order allow,deny
-		Allow from all
-	</Directory>
-	
-	php_value include_path .:/usr/share/php:$pathToPear:/Users/$username/Sites/loris/project/libraries:/Users/$username/Sites/loris/php/libraries:/usr/local/lib/php/Smarty
-	# Possible values include: debug, info, notice, warn, error, crit,
-	# alert, emerg.
-	LogLevel warn
-
-	ServerSignature Off
-
-	#SSLEngine Off  # change to On to enable, after updating cert paths below
-	#SSLCertificateFile /etc/apache2/ssl/loris-cert.pem
-	#SSLCertificateKeyFile /etc/apache2/ssl/loris-key.pem
-	#SSLCACertificateFile /etc/apache2/ssl/CA-cacert.pem
-</VirtualHost>
 
-<VirtualHost *:80>
-	ServerName localhost
-	DocumentRoot /Library/WebServer/Documents/
-</VirtualHost>
+## 2.0 Install PHP extension: ext-ast ^0.1.5 for phan/phan
+Type into your terminal and hit enter:
+```
+pecl install ast-0.1.5;
 ```
- * <i>$username ⇾ your username for the computer</i>
- * <i>$pathToPear ⇾ path where PEAR installs .php files (see [section 1.3.4](#include_path) part 1)</i>
 
-## 3.6 Update Database
+## 2.1 Install Node.js
+Navigate in your browser to [NodeJS.org](https://nodejs.org/en/) to download and run the installer.
 
-Note that the default Loris setup assumes that Loris is running on localhost. If this
-is not the case, you'll have to manually update the URL and Host config variables in the
-ConfigSettings table by running the following SQL commands from a MySQL prompt:
 
-```SQL
-UPDATE Config SET Value='$yourURL' WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='url');
-UPDATE Config SET Value='$yourHostname' WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='host');
+## 2.2 Install Loris
+Paste into your terminal and hit enter:
+```
+cd ~/Development/GitHub/Loris
+make dev
+cd /tools
+./install.sh
 ```
 
-Make sure that `$yourURL` above contains the "http://" or "https://" and `$yourHostname` does not. If your server is only being accessed from localhost, you can skip this step.
+## 2.2.1 Launch the Loris installer in your browser
+Open a browser (we officially support Firefox & Chrome) and navigate to:
+```
+localhost:8080/installdb.php
```

Comment:
instruction to install php specify using port 9000.  Apache use port 80 (443 with SSL) by default. 
Better specify a commons port number or add instruction on how to decide which port number to use

Reply:
@PapillonMcGill thanks for the review changes! The Apache setup while using brew will default to port 8080. I'm not sure if it's necessary to include more since this is a development setup. I previously had instructions to change it to port 80 (I believe) but @driusan thought it wouldn't be best compared to leaving it the default.